cockpit/lib/cockpit-components-dialog.jsx

```diff
@@ -45,86 +47,111 @@ let _ = cockpit.gettext;
  *  - idle_message optional, always show this message on the last row when idle
  *  - dialog_done optional, callback when dialog is finished (param true if success, false on cancel)
  */
-class DialogFooter extends React.Component {
+export class DialogFooter extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
             action_in_progress: false,
             action_in_progress_promise: null,
             action_progress_message: '',
+            action_progress_cancel: null,
             action_canceled: false,
             error_message: null,
         };
+        this.keyUpHandler = this.keyUpHandler.bind(this);
+        this.update_progress = this.update_progress.bind(this);
+        this.cancel_click = this.cancel_click.bind(this);
     }
+
     keyUpHandler(e) {
-        if (e.keyCode === 27) {
+        if (e.keyCode == 27) {
```

Comment:
Why use loose equality here?

Reply:
Why not? :-)

`cockpit-component-dialog.jsx` in the Cockpit sources always has used `==` here.  The copy here got changed in 9c7c1d344628f0449ddcf1a263e2e4f80e62219d.

There are quite a number of changes like this, like changing `cancel_click(e)` to `cancel_click(event)`.  Do you want me to keep them?

Reply:
This commit has been merged as #2195